src/plugins/kibana/server/lib/ingest/pipeline/__tests__/converter.js

```diff
@@ -0,0 +1,68 @@
+import expect from 'expect.js';
+import sinon from 'sinon';
+import _ from 'lodash';
+import { kibanaToEs, esToKibana } from '../converter';
+import setConverter from '../../processors/set/converter';
+import joinConverter from '../../processors/join/converter';
+
+describe('ingest', () => {
+
+  describe('pipeline', () => {
+
+    describe('converter', () => {
+
+      describe('kibanaToEs', () => {
+
+        let source;
+        let expected;
+        let setStub;
+        let joinStub;
+        beforeEach(function () {
+          setStub = sinon.stub(setConverter, 'kibanaToEs', () => {
+            return { foo: 'bar' };
+          });
+          joinStub = sinon.stub(joinConverter, 'kibanaToEs', () => {
+            return { bar: 'baz' };
+          });
+
+          source = [ { type_id: 'set' }, { type_id: 'join' }];
+          expected = {
+            processors: [ { foo: 'bar' }, { bar: 'baz' } ]
+          };
+        });
+
+        afterEach(function () {
+          setConverter.kibanaToEs.restore();
+          joinConverter.kibanaToEs.restore();
```

Comment:
For future reference, I believe you can avoid these if you import `auto-release-sinon` rather than `sinon` directly, for example: https://github.com/elastic/kibana/blob/master/src/ui/public/courier/fetch/__tests__/fetch.js#L3